aospy/model.py

```diff
@@ -14,36 +15,33 @@
 from .utils.times import datetime_or_default
 from . import utils
 
+# Need to set dask to use the serial scheduler in order to use multiprocess
+# to compute calculations in parallel
+dask.set_options(get=dask.async.get_sync)
+
 
 class Model(object):
     """Parameters of local data associated with a single climate model."""
     def __init__(self, name='', description='', proj=None,
-                 grid_file_paths=None, data_direc=None,
-                 data_dir_struc=None, data_dur=None,
-                 data_start_date=None, data_end_date=None,
-                 default_start_date=None, default_end_date=None,
+                 grid_file_paths=None, default_start_date=None,
+                 default_end_date=None,
                  runs={}, default_runs={},
-                 load_grid_data=False, repo_version=None,
-                 repo_ens_mem='r1i1p1'):
+                 load_grid_data=False,
+                 data_loader=None):
         self.name = name
         self.description = description
         self.proj = proj
 
         grid_file_paths = [] if grid_file_paths is None else grid_file_paths
         self.grid_file_paths = grid_file_paths
-        self.repo_version = repo_version
-        self.repo_ens_mem = repo_ens_mem
-
-        self.data_direc = data_direc
-        self.data_dir_struc = data_dir_struc
-        self.data_dur = data_dur
 
-        self.data_start_date = datetime_or_default(data_start_date, None)
-        self.data_end_date = datetime_or_default(data_end_date, None)
-        self.default_start_date = datetime_or_default(default_start_date,
-                                                      self.data_start_date)
-        self.default_end_date = datetime_or_default(default_end_date,
-                                                    self.data_end_date)
+        try:
```

Comment:
The issue is that not all DataLoaders necessarily need ```data_start_date``` and ```data_end_date``` attributes.  On revisiting this it occurs to me one can obtain the desired behavior here more elegantly (and transparently) with ```getattr```, with a default value of ```None```.